app/src/main/java/org/linphone/contacts/ContactsFragment.java

```diff
@@ -636,7 +642,9 @@ public void onResponse(Call<ContactList> call, Response<ContactList> response) {
                                 mContactsRefresher.setRefreshing(false);
 
                                 mContactAdapter.notifyDataSetChanged();
-                                isLoading = false;
+
+                                scrollToBottom(mContactsList);
```

Comment:
Non ho capito questa chiamata a scroll to bottom. Se uso l'app su uno smartphone e scrollo sembra comportarsi in modo strano.

Reply:
Non più necessario dopo la modifca alle API